geoplateforme/api/processing.py

```diff
@@ -64,6 +64,44 @@ def get_base_url(self, datastore_id: str) -> str:
         """
         return f"{self.plg_settings.base_url_api_entrepot}/datastores/{datastore_id}/processings"
 
+    def get_processing_by_id(
+        self, datastore_id: str, possible_ids: list[str]
+    ) -> Processing:
+        """Get processing from id.
+
+        :param datastore_id: datastore id
+        :type datastore_id: str
+        :param possible_ids: possible ids for processing
+        :type name: list[str]
+
+        :raises UnavailableProcessingException: when error occur during requesting the API
+
+        :return: processing
+        :rtype: Processing
+        """
+        self.log(
+            f"{__name__}.get_processing(datastore:{datastore_id},possible_ids:{possible_ids})"
```

Comment:
le nom de la fonction n'est pas bon
```suggestion
            f"{__name__}.get_processing_by_id(datastore:{datastore_id},possible_ids:{possible_ids})"
```